### DISCUSSION TITLE
Cannot access to swagger UI

### DISCUSSION THREAD

**Discussion by minuth** [runsidekick/sidekick#51] [Q&A] [answered]
I cannot access to swagger UI after running the docker image successfully.
<img width="1082" alt="image" src="https://user-images.githubusercontent.com/33181465/196879355-c619e3bf-ad2f-495a-8a0d-7e2541dfc5d1.png">

How can I solve it?

**Comment by boroskoyo** [comment]
hi minuth, thanks for reaching out. can you make sure you have the latest version? 
you can also try the old swagger URL, which was http://localhost:8084/swagger-ui.html before the swagger update.

**Reply by minuth** [reply]
Yes, it is the latest version. This URL http://localhost:8084/swagger-ui.html doesn’t work too.

**Comment by boroskoyo** [comment]
also you can check out https://api.service.runsidekick.com/swagger-ui/index.html if you're working with the Sidekick Cloud

**Comment by minuth** [comment]
I also cannot run the quick start project.  seems like it cannot connect to the docker services.
![image](https://user-images.githubusercontent.com/33181465/196891369-82d965f9-3ead-479e-bd1e-afb9aca606a4.png)

**Reply by boroskoyo** [reply]
sounds like a configuration issue. can you make sure your  apikey, host & port  properties. your apikey should be same as the one you set here https://github.com/runsidekick/sidekick#running-sidekick-using-our-docker-image
```
sidekickDebugger.start({
  apiKey:"my-token", 
  SIDEKICK_AGENT_APPLICATION_NAME: "sidekick-demo-app",
  SIDEKICK_AGENT_LOG_LEVEL:"debug",
  SIDEKICK_AGENT_BROKER_HOST: "ws://127.0.0.1",
  brokerPort:7777,
  applicationVersion:"1.0",
  applicationStage :"lab"
});

```

we can connect on discord if your issue persists: https://www.runsidekick.com/discord-invitation

**Reply by minuth** [reply]
The configuration is correct. I didn't change anything just clone the project and run.

**Comment by boroskoyo** [comment, marked as answer]
Seems like our docker image does not work on arm based machines. We will be updating our README to include information about how you can build Sidekick on your arm devices.

**Reply by boroskoyo** [reply]
You can follow Build Service Images (Advanced) section in the README to build the images for yourself
